packages/network/src/chart/Network.jsx

```diff
@@ -65,6 +65,7 @@ class Network extends React.PureComponent {
       this.props.graph.links !== graph.links
```

Comment:
what do you think about making this `if (this.state.computingLayout || this.props.graph.nodes !== graph.nodes || this.props.links !== graph.links)` since `if` and `else if` blocks currently do almost the same thing? (I don't think there's harm in setting `{ computingLayout: true }` if it's already `true`)